nipoppy/trackers/run_tracker.py

```diff
@@ -161,19 +161,23 @@ def run(global_configs, dash_schema_file, pipelines, session_id="ALL", run_id="1
 
                 if pipeline == "heudiconv":
                     subject_dir = f"{DATASET_ROOT}/bids/{bids_id}"
-                elif pipeline == "freesurfer":
+                    subject_ses_dir = f"{subject_dir}/{session}"
+                elif pipeline in ["freesurfer","tractoflow"]:
                     subject_dir = f"{DATASET_ROOT}/derivatives/{pipeline}/v{version}/output/{session}/{bids_id}" 
+                    subject_ses_dir = subject_dir
                 elif pipeline in BIDS_PIPES:
                     subject_dir = f"{DATASET_ROOT}/derivatives/{pipeline}/v{version}/output/{bids_id}" 
+                    subject_ses_dir = f"{subject_dir}/{session}"
                 elif pipeline in NO_TRACKER_PIPES:
                     logger.warning(f"pipeline: {pipeline} does not have a tracker yet...")
                 else:
                     logger.error(f"unknown pipeline: {pipeline}")
                     
-                dir_status = Path(subject_dir).is_dir()
-                logger.debug(f"subject_dir:{subject_dir}, dir_status: {dir_status}")
+                subject_ses_dir_status = Path(subject_ses_dir).is_dir()
+                logger.debug(f"subject_dir:{subject_ses_dir_status}, dir_status: {subject_ses_dir_status}")
                 
-                if dir_status:                                 
+                # TODO incorporate manifest datatype availability
```

Comment:
Are we going to use the manifest here *instead* of the doughnut or *in addition* to the doughnut? I feel like it would replace the doughnut since the doughnut tells us who was successfully converted to BIDS, but we'll just use `INCOMPLETE` for people who haven't been successfully converted to BIDS (if I understand correctly). If we don't want to differentiate between "no BIDS data yet" and "with BIDS but not run" then we only need the manifest I think

Reply:
Also, the `datatype` for the manifest only gives `anat`/`func`/`dwi`/etc. level of information, it doesn't say if someone has `anat` but only T2w (which is a case in PPMI...). So it might be a little accurate for these edge cases but I think that should still be fine (?)

Reply:
All valid questions which basically made me procrastinate on making these changes until recently :) 

But yes, I think only using `manifest` as expected truth makes sense to me. `doughnuts` to me are more or less are like `bagels` informing disk status of organized raw data. 

Now that we have `bids_tracker`, we can assign `INCOMPLETE` for a `participant x session`, if the manifest specifies a `datatype` for it but no corresponding directory is found on disk. Along with pipeline trackers, this would help pinpoint what "expected" outputs are incomplete in general. 

Does this make sense?

Reply:
Yes I think so!